api/query/autocomplete.go

```diff
@@ -123,7 +123,7 @@ func prepareAutocompleteResponse(limit int, filters *shared.QueryFilter, testRun
 		}
 	}
 
-	files := make([]AutocompleteResult, 0, fileSet.Cardinality()/len(testRuns))
+	var files []AutocompleteResult
```

Comment:
Hmm, why is this changed?

Reply:
Drive-by fix for 500 when no test runs are found for an autocomplete query.

Reply:
Hmm, sorry could you explain what was wrong with the original statement?

Reply:
`len(test runs)` is zero when no test runs get loaded. Division by zero is bad.

Reply:
Ahh I see. Thanks!